coding/python_style_guide.rst

```diff
@@ -469,22 +469,22 @@ This comment should appear after the ``def`` line.
 We follow `PEP 8ʼs naming conventions <https://www.python.org/dev/peps/pep-0008/#naming-conventions>`_, with exceptions listed here.
 The naming conventions for LSST Python and C++ source have been defined to be as similar as the respective languages allow.
 
-In general:
+All LSST Python source code is consistent with PEP-8 naming as follows:
 
 - class names are ``CamelCase`` with leading uppercase,
 - module variables used as module global constants are ``UPPERCASE_WITH_UNDERSCORES``,
-- all other names are ``camelCase`` with leading lowercase.
```

Comment:
Oh I understand what happened to my earlier list. Here's the last item. Generally it's awkward to have a 1-item list.

Maybe splitting off Science Pipelines Naming Conventions into a following section would help here.